app/routes.py

```diff
@@ -1,6 +1,6 @@
-from app import app
+from app import app, db
 from flask import jsonify, request
-from app.models import User, Post
+from app.models import User, Post 
```

Comment:
There is a space at the end of the line. You can install an extension on vs code to automatically format (have correct identation, spaces between modules etc) your python code every time you save. The python module which takes care of the automatic format of your code is called `black`. Follow this link to install it on vs code https://dev.to/adamlombard/how-to-use-the-black-python-code-formatter-in-vscode-3lo0